index.js

```diff
@@ -30,7 +30,14 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+// use map to loop though
+// returns all titles from movies
+// if empty, throw error message
+
+function getAllMovieTitles(movies) {
+  if (!movies.length) {throw `There are no movies`}
```

Comment:
Try to follow the rule that `if` and `else` statements should have code blocks even for single statements.